### PR TITLE
Complaint tool issue info

### PR DESCRIPTION
**Pull request by rrmatthews**
These changes set up the applicant information page for the complaint tool as described by this [ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11578)

## TODO:
- [x] Write schema / uiSchema
- [x] Write unit tests
- [ ] Verify content

## Screenshots: 
localhost_3001_education_complaint-tool_form_applicant-information 1

### CODE REVIEW

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu Thanks.. Please see below for a couple of changes from me: 

We need to replace "Complaint" with "Feedback" in the  breadcrumb

Please fix the misspelling of Feedback in the header Feedback Information

per the prototype the question should read:
Which topic best describes your feedback? (Select all that apply) (*Required)

We need to add two more issues to the list: 
--Transfer of credits
--Refund issues

it looks like the character count info is missing from the two questions at the bottom. Do we no longer have a limit? 

Please rewrite the first question at the bottom TO:
Please write your feedback and any details about your issue in the space below. 

--for the bullet list, we are still waiting on copy/examples  of each topic from the stakeholders.

**Comment by rrmatthews** [comment]
I noted that the issue description character limit is set a 32000 on the backend, do we want to mention this to the user (instead of the current 1000 character limit)? @lihanli

**Comment by rrmatthews** [comment]
Thanks @peggygannon , made the changes requested and updated the screenshot

**Review by cehsu — APPROVED** [review]
LGTM, pending @peggygannon's approval. We will want to merge this after the applicant information page PR, since this PR includes those changes (I have updated the applicant information changes to be in sync with the latest version of that PR). I removed the contact information changes, so that the contact information PR would not block this one.

**Review by rianfowler — APPROVED** [review]

**Comment by peggygannon** [comment]
@cehsu @rrmatthews I don't see the updated screenshot above with the changes..

**Comment by rrmatthews** [comment]
@peggygannon this should be the screenshot

43284678-971e4504-90ea-11e8-910f-79070a10f86e

**Comment by lihanli** [comment]
@rrmatthews yea the limit description should match the limit on the schema

**Comment by rrmatthews** [comment]
@lihanli okay thank you, made that change. @peggygannon screenshot has been updated, but please note that the issue description character limit says 32,000 to match the backend limit as well

**Comment by peggygannon** [comment]
@cehsu @rrmatthews The following changes aren't reflected in the updated screenshot: 

We need to replace "Complaint" with "Feedback" in the breadcrumb

Please fix the misspelling of Feedback in the header "Feedback Information"

per the prototype the question should read:
Which topic best describes your feedback? (Select all that apply) (*Required)

Please fix misspelling of "characters" in both instances. 

Please rewrite the first question at the bottom TO:
"Please write your feedback and any details about your issue in the space below."

**Comment by cehsu** [comment]
Thanks @peggygannon, kindly refer to the updated screenshot below.
localhost_3001_education_complaint-tool_form_feedback-information 1

File: package.json
Comment (cehsu) [inline package.json]: The latest version is now: 444e3aba11ac1ce27e869d24bbee20160bfa3ea6, let's make sure when we merge this that the diff indicates that we are updating the version from 4e5d6a1082e8268b79cb3c433ba379ce97343bc0 and not switching it back from 444e3aba11ac1ce27e869d24bbee20160bfa3ea6 if it has already been updated by another PR.

**Review by peggygannon — APPROVED** [review]
@cehsu Complaint still needs to be removed from breadcrumb.

Should the second text box also all for 32,000 characters? If so, we need to replace 1,000 with 32,000..

**Comment by cehsu** [comment]
Thanks again @peggygannon, updated screenshot provided below. The two fields have different word limits, that is why the limit provided to the user is different.
localhost_3001_education_complaint-tool_form_applicant-information 3

**Comment by saintsoup52** [comment]
FWIW There is a missing category of complaints in the screen shots above... "Other"

**Pull request merged**
